src/components/ProfileGamesTable.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+
+import Paper from "@material-ui/core/Paper";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableContainer from "@material-ui/core/TableContainer";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import StarIcon from "@material-ui/icons/Star";
+import amber from "@material-ui/core/colors/amber";
+import grey from "@material-ui/core/colors/grey";
+import moment from "moment";
+
+import ElapsedTime from "./ElapsedTime";
+import User from "./User";
+import Loading from "./Loading";
+
+const useStyles = makeStyles((theme) => ({
+  gamesTable: {
+    display: "flex",
+    flexDirection: "column",
+    maxHeight: 400,
+    marginBottom: theme.spacing(1),
+    whiteSpace: "nowrap",
+    "& td, & th": {
+      paddingTop: 6,
+      paddingBottom: 6,
+      paddingLeft: 12,
+      paddingRight: 12,
+    },
+    "& svg": {
+      display: "block",
+    },
+    "& tr:hover": {
+      background: theme.palette.action.hover,
+      cursor: "pointer",
+    },
+  },
+  // Remove cells of some columns of table for small screens
+  vanishingTableCell: {
+    [theme.breakpoints.down("xs")]: {
+      display: "none",
+    },
+  },
+}));
+function ProfileGamesTable({ userId, gamesData, handleClickGame }) {
+  const classes = useStyles();
+  if (!gamesData) {
+    return <Loading />;
+  }
+  if (Object.keys(gamesData).length === 0) {
+    return (
+      <Typography style={{ color: grey[300] }}>
+        No games finished yet...
+      </Typography>
+    );
+  }
+  return (
+    <TableContainer component={Paper} className={classes.gamesTable}>
+      <Table size="small" stickyHeader>
+        <TableHead>
+          <TableRow>
+            <TableCell>#</TableCell>
```

Comment:
Do we really need to keep a column that just counts up from 1 to n? It seems distracting.